lib/Service/RoomService.php

```diff
@@ -76,6 +82,12 @@ public function findByUserId(string $userId): array {
 	public function create(string $name, string $welcome, int $maxParticipants, bool $record, string $access, string $userId): \OCP\AppFramework\Db\Entity {
 		$room = new Room();
 
+		if($this->config->getAppValue('bbb', 'join.defaultMedia') === 'true') {
+			$media = false;
+		} else {
+			$media = true;
+		}
+
```

Comment:
Can you rewrite this to `$mediaCheck = $this->config->getAppValue('bbb', 'join.defaultMedia') !== 'true';`.

I understand why you used `join.defaultMedia` as parameter, but if you read the code you have to think twice to understand why the media check is false if the default media is true. For me it would be easier if the option would be renamed to `join.mediaCheck`.

Reply:
Thanks for the quick feedback.
Just for my understanding - you want me to change the following two variable names, correct?

$media -> $mediaCheck
join.defaultMedia -> join.mediaCheck

Reply:
Yes that's right. I think it's more consistent. Otherwise you have three variable names for the same thing.

Reply:
This should resolve the requested changes :D

Reply:
We are close :joy:

If `join.mediaCheck` is `true`, `$checkMedia` is `false`. Do you have time to change that, or should I jump in?

Reply:
Feel free to jump in 😂

Reply:
Hey @sualko,
 
I have checked the current state of the PR - on my test system there were no issues. 
Could you point me to what you think I should change 😅

Reply:
Thanks for your reminder. This week a project is ending for me and a new one is starting therefore I had no time to adjust this pr. It just about the readability:

```
$mediaCheck = $this->config->getAppValue('bbb', 'join.mediaCheck') !== 'true';
```
If `join.mediaCheck` is true, `$mediaCheck` is false. This is confusing as a dev. Therefore we should change the checkbox label to change the behaviour.

Reply:
No problem - I have been busy as well ;D

I have changed the label of the checkbox, so that the admin now decides whether the mediaCheck will be performed or not (instead of whether it will be skipped or not). `join.mediaCheck` and `$mediaCheck` can therefore now be the same value.

Hope that I understood it correctly and fixed it with the last Commit.